homeassistant/components/hive/config_flow.py

```diff
@@ -145,12 +144,12 @@ async def async_setup_hive_entry(self) -> ConfigFlowResult:
         # Setup the config entry
         self.data["tokens"] = self.tokens
         if self.source == SOURCE_REAUTH:
```

Comment:
A little refactoring here - to align with more recent implementations (use of `self._get_reauth_entry()`)